src/mame/drivers/myb3k.cpp

```diff
@@ -77,173 +91,426 @@ class myb3k_state : public driver_device
-	DECLARE_WRITE_LINE_MEMBER( dack3_w ) { LOGDMA("%s: %d\n", FUNCNAME, state); select_dma_channel(3, state); }
-	DECLARE_READ8_MEMBER(dma_memory_read_byte);
-	DECLARE_WRITE8_MEMBER(dma_memory_write_byte);
-	DECLARE_WRITE8_MEMBER(dma_segment_w);
+	/* DMA controller */
 	DECLARE_WRITE_LINE_MEMBER( hrq_w );
 	DECLARE_WRITE_LINE_MEMBER( tc_w );
+	DECLARE_WRITE8_MEMBER(dma_segment_w);
+	DECLARE_READ8_MEMBER(dma_memory_read_byte);
+	DECLARE_WRITE8_MEMBER(dma_memory_write_byte);
+	DECLARE_READ8_MEMBER( io_dack0_r )  { uint8_t tmp = m_isabus->dack_r(0); LOGDMA("%s: %02x\n", FUNCNAME, tmp); return tmp; }
+	DECLARE_READ8_MEMBER( io_dack1_r )  { uint8_t tmp = m_isabus->dack_r(1); LOGDMA("%s: %02x\n", FUNCNAME, tmp); return tmp; }
+	DECLARE_READ8_MEMBER( io_dack2_r )  { uint8_t tmp = m_isabus->dack_r(2); LOGDMA("%s: %02x\n", FUNCNAME, tmp); return tmp; }
+	DECLARE_READ8_MEMBER( io_dack3_r )  { uint8_t tmp = m_isabus->dack_r(3); LOGDMA("%s: %02x\n", FUNCNAME, tmp); return tmp; }
+	DECLARE_WRITE8_MEMBER( io_dack0_w ) { LOGDMA("%s: %02x\n", FUNCNAME, data); m_isabus->dack_w(0,data); }
+	DECLARE_WRITE8_MEMBER( io_dack1_w ) { LOGDMA("%s: %02x\n", FUNCNAME, data); m_isabus->dack_w(1,data); }
+	DECLARE_WRITE8_MEMBER( io_dack2_w ) { LOGDMA("%s: %02x\n", FUNCNAME, data); m_isabus->dack_w(2,data); }
+	DECLARE_WRITE8_MEMBER( io_dack3_w ) { LOGDMA("%s: %02x\n", FUNCNAME, data); m_isabus->dack_w(3,data); }
+	DECLARE_WRITE_LINE_MEMBER( dack0_w ){ LOGDMA("%s: %d\n", FUNCNAME, state); select_dma_channel(0, state); }
+	DECLARE_WRITE_LINE_MEMBER( dack1_w ){ LOGDMA("%s: %d\n", FUNCNAME, state); select_dma_channel(1, state); }
+	DECLARE_WRITE_LINE_MEMBER( dack2_w ){ LOGDMA("%s: %d\n", FUNCNAME, state); select_dma_channel(2, state); }
+	DECLARE_WRITE_LINE_MEMBER( dack3_w ){ LOGDMA("%s: %d\n", FUNCNAME, state); select_dma_channel(3, state); }
+
+	/* Timer */
+	DECLARE_WRITE_LINE_MEMBER( pit_out1_changed );
+
+	/* Video controller */
+	MC6845_UPDATE_ROW(crtc_update_row);
+
+	/* ISA+ Expansion bus */
+	DECLARE_WRITE_LINE_MEMBER( isa_irq5_w );
+	DECLARE_WRITE_LINE_MEMBER( isa_irq7_w );
+
+	/* Centronics  */
+	DECLARE_WRITE_LINE_MEMBER (centronics_ack_w);
+	DECLARE_WRITE_LINE_MEMBER (centronics_busy_w);
+	DECLARE_WRITE_LINE_MEMBER (centronics_perror_w);
+	DECLARE_WRITE_LINE_MEMBER (centronics_select_w);
+
+	/* Keyboard */
+	DECLARE_READ8_MEMBER(myb3k_kbd_r);
+	void kbd_set_data_and_interrupt(u8 data);
+
+	/* Video Controller */
+	DECLARE_WRITE8_MEMBER(myb3k_video_mode_w);
+
+	/* Monitor */
+	DECLARE_INPUT_CHANGED_MEMBER(monitor_changed);
+
+	/* Status bits */
+	DECLARE_READ8_MEMBER(myb3k_io_status_r);
+
+private:
+	virtual void machine_start() override;
+	virtual void machine_reset() override;
+
+	/* Interrupt Controller */
+	void pic_ir5_w(int source, int state);
+	void pic_ir7_w(int source, int state);
+
+	/* Jumper fields J4/J5 */
+	enum
+	{
+		PPI_PC3  = 0x01,
+		ISA_IRQ5 = 0x02,
+		ISA_IRQ7 = 0x04,
+		CENT_ACK = 0x08
+	};
+
+	/* Paralell port */
+	enum {
```

Comment:
This is inconsistent with the other `enum` declarations (opening brace on same line vs next line).